doc/user_manual/PostProcessors/Validation.tex

```diff
@@ -105,7 +108,19 @@ \subsubsection{Validation PostProcessors}
     refer to \ref{sec:Metrics} for detailed descriptions about this metric.
     \item \xmlNode{pivotParameterFeature}, \xmlDesc{string, required field}, specifies the pivotParameter for a feature <HistorySet>. The feature pivot parameter is the shared index of the output variables in the data object.
     \item \xmlNode{pivotParameterTarget}, \xmlDesc{string, required field}, specifies the pivotParameter for a target <HistorySet>. The target pivot parameter is the shared index of the output variables in the data object.
-    \item \xmlNode{multiOutput}, \xmlDesc{string, required field}, to extract raw values for the HistorySet. The user must use ‘raw values’ for the full set of metrics’ calculations to be dumped.
+    \item \xmlNode{separateFeatureData}, \xmlDesc{string, optional field}, specifies the custom feature interval to apply DSS postprocessing. The string should contain three parts; start time, `|', and end time all in one. For example, 0.0|0.5.
+      The start and end time should be in ratios or raw values of the full interval. In this case 0.5 would be either the midpoint time or time 0.5 of the given time units. This is node is not required and if not provided, the default is the full time interval.
+      the following attributes need to be specified:
+      \begin{itemize}
+        \item \xmlAttr{type}, \xmlDesc{required string attribute}, options are `ratio' or `raw\_values'. The default is `ratio'.
+      \end{itemize}
+    \item \xmlNode{separateTargetData}, \xmlDesc{string, optional field}, specifies the custom target interval to apply DSS postprocessing. The string should contain three parts; start time, `|', and end time all in one. For example, 0.0|0.5.
+      The start and end time should be in ratios or raw values of the full interval. In this case 0.5 would be either the midpoint time or time 0.5 of the given time units. This is node is not required and if not provided, the default is the full time interval.
+      the following attributes need to be specified:
+      \begin{itemize}
+        \item \xmlAttr{type}, \xmlDesc{required string attribute}, options are `ratio' or `raw\_values'. The default is `ratio'.
+      \end{itemize}
+    \item \xmlNode{multiOutput}, \xmlDesc{string, required field}, to extract raw values for the HistorySet. The user must use `raw values' for the full set of metrics’ calculations to be dumped.
```

Comment:
Do you have plan to add different options for "multiOutput"? If not, it seems you do not need to expose this node to user, you can directly define the value for multiOutput in the init method of the DSS class.